data/ignition/files/usr/local/share/assisted-service/pod.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: assisted-installer
+  name: assisted-installer
+spec:
+  containers:
+  - args:
+    - run-postgresql
+    image: quay.io/centos7/postgresql-12-centos7:latest
```

Comment:
This is OK for now, but eventually we'll need to do something other than CentOS 7.

Reply:
Eventually we'll need all of these images in the release payload.
Just realised that this means postgres as well :scream_cat:

Reply:
Can we avoid that? This thing only has to support 1 cluster's worth of nodes, which means at most a couple of hundred. Do we even need a database for that?

Reply:
@pawanpinjarkar Would know more about the challenges up moving up the database version. I think there are some things that would need to be updated. Didn't we find that out with moving the unit tests to prow?

Reply:
With that said we should absolutely look at least moving up the version of Postgres. SQLite has been looked at in the past I don't know the full context there. We should tie in the rest of the Assisted Installer team for this.

Reply:
The database version was moved from `quay.io/ocpmetal/postgres:12.3-alpine` to `quay.io/ocpmetal/postgresql-12-centos7` and the reason at that time was to use only one consistent image. The `quay.io/ocpmetal/postgresql-12-centos7` image was chosen which is same as `quay.io/edge-infrastructure/postgresql-12-centos7:latest`

As we already have `quay.io/ocpmetal/postgresql-12-centos7` image available for CI [here](https://github.com/openshift/release/blame/master/clusters/app.ci/supplemental-ci-images/ocpmetal-postgresql-12-centos7.yaml), with the unit test task we decided to use this centos 7 image and use edge-infrastructure instead of ocpmetal.

We may use the upstream image `quay.io/centos7/postgresql-12-centos7:latest` as used in the podman flow in assisted-service. Ref: https://github.com/openshift/assisted-service/pull/3110

Reply:
I also think we can go with the next version from upstream i.e. `quay.io/centos7/postgresql-13-centos7:latest`

Reply:
I think using centos:stream probably makes more sense. The database removal is a bit of a stretch goal.

Reply:
I think there was something about goarm compatibility when we discusses use sqllite for testing. But I might be wrong.

Reply:
> I think using centos:stream probably makes more sense. The database removal is a bit of a stretch goal.

We can't put a CentOS image into the release payload, so at some point that image is going to have to be replaced with one based on something we can ship in the product.